src/components/pages/Analytics/TechStackGraph.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from 'react';
+import embed from 'vega-embed';
+export default async function TechStackGraph() {
+  let url = `${process.env.REACT_APP_DS_API_URL}/graph/tech-stack-by-role`;
+  // embed is an asynchronous api call, resolves in a promise
+  // for more information visit https://github.com/vega/vega-embed
+  useEffect(() => {
```

Comment:
Great idea to use vegas embed functionality inside of useEffect. The code seems a lot cleaner this way.